jobs/feeds.py

```diff
@@ -10,24 +10,23 @@
 
 @shared_task
 def get_feeds():
-    commands = []
     all_venues = Venues.objects.all()
     check_and_stop_running_processes()
 
     for venue in all_venues:
         venue_name = venue["venue_name"]
```

Comment:
As venue_name is used to construct a path name on the filesystem, best to apply a strict regexp to it e.g. A-Za-z0-9 and ignore venues that don't pass the regexp.